app/models/favorite.rb

```diff
@@ -6,11 +6,17 @@ def initialize(initial_contents)
   end
 
   def add_pet(pet)
-    @contents << pet
+    @contents << pet.id
   end
 
   def total_count
     @contents.count
   end
 
+  def pets_by_id
+    @contents.collect do |id|
```

Comment:
Is this an active record method?

Reply:
I don't know if it is or not, but I tried .each and it didn't work so I started trying other Ruby methods.

Reply:
I'm specifically referring to the `.collect` method.

Reply:
Oh! It's an enumerable, I thought it was an Active Record method.  Thanks for clarifying!